javascript/src/com/google/idea/blaze/javascript/run/producers/NonBlazeProducerSuppressor.java

```diff
@@ -40,6 +39,9 @@ public class NonBlazeProducerSuppressor implements StartupActivity {
       PRODUCERS_TO_SUPPRESS =
           ImmutableList.of(
               JestRunConfigurationProducer.class,
+              // Not suppressing JestRunConfigurationProduce since that prevents Jest tests from
+              // running properly, see https://github.com/bazelbuild/intellij/issues/3629
+              // JestRunConfigurationProducer.class,
```

Comment:
I think you forgot to remove it here:
```suggestion
              // Not suppressing JestRunConfigurationProduce since that prevents Jest tests from
              // running properly, see https://github.com/bazelbuild/intellij/issues/3629
              // JestRunConfigurationProducer.class,
```